base/show.jl

```diff
@@ -322,6 +327,11 @@ function isidentifier(s::AbstractString)
     end
     return true
 end
+"""
+    isidentifier(s) -> Bool
+
+Returns whether symbol is vaild identifier.
```

Comment:
Okay, good to know. Still -- identifiers exist in source code, and symbols exist in Julia internally. `isidentifier` makes most sense when parsing source code, to distinguish between words that are identifiers or e.g. numbers (or symbols).

Once you already have a symbol, I assume you can call `isidentifier` to check whether this is an internally-generated symbol or not. I missed this part in my explanation above. I still would explain the string/identifier connection first since it's easier to understand.